features/FEATURE_BLE/targets/TARGET_WISE1530_F412RE/HCIDriver.cpp

```diff
@@ -0,0 +1,428 @@
+#include <stdio.h>
+#include "CordioBLE.h"
+#include "CordioHCIDriver.h"
+#include "mbed.h"
```

Comment:
what is needed here from mbed.h ? DigitalInout and PinNames ?

 missing license header file

Reply:
looks like it was not used